### ISSUE TITLE
Page Builder not accessible

### ISSUE THREAD

**Issue by ganeddact** [magento/magento2-page-builder#897]
On June 28th, the European Accessibility Act (EAA) comes into force.
There are several issues with the accessibility of pages once they are built with Page Builder.

**Carousel**
Take for example the template 
app/code/Magento/PageBuilder/view/frontend/templates/catalog/product/widget/content/carousel.phtml

Why does it have a ol tag that contain elements that are not li?
As you can see here:
https://dequeuniversity.com/rules/axe/4.4/list?application=axeAPI
ol elements should only contain li, script or template elements.

**Row and columns**
If you create a page with rows and columns to create a sort of list of elements with a call to action, they are rendered as a list of divs, making it difficult for assistive technology to communicate the relationships to users. There should be an option to differentiate if the content is a sort of list or if it's not related to each other, and act accordingly with the html generation.
